android/appv2/build.gradle.kts

```diff
@@ -65,6 +65,9 @@ dependencies {
     implementation(libs.ui.graphics)
     implementation(libs.compose.ui.tooling.preview)
     implementation(libs.compose.material3)
+    implementation(project(":data:event"))
+    implementation(project(":data:local-sqldelight"))
+    implementation(project(":data:match"))
```

Comment:
Not 100% sure if we need these, we can probably put the DI code in shared somehow.